src/Control/Effect/Lift.hs

```diff
@@ -1,11 +1,12 @@
 {- | Provides a mechanism to kick off the evaluation of an effect stack that takes place in a monadic context.
 
-'Lift' effects are always the last effect in a given effect stack. These stacks are invoked with 'Control.Effect.Lift.runM'. The 'Control.Effect.Pure.Pure' effect is equivalent to @Lift Identity@.
+'Lift' effects are always the last effect in a given effect stack. These stacks are invoked with 'Control.Carrier.Lift.runM' or 'Control.Algebra.run'.
```

Comment:
1. I noticed the final sentence (added by @patrickt in #251) yesterday, and it blew my mind. I don’t know how I’d never realized this before, but yes, `Lift Identity` _is_ an equivalent terminal effect for pure code. `sendM (Identity x)` isn’t the most readable synonym for `pure x`, but by contrast, `Pure` doesn’t offer any operations at all—making it kind of a degenerate edge case.

2. I love that the first sentence is now true in two ways: if `Lift` occurs, it’s the final effect; and `Lift` now occurs in all (ok, _most_, since #206) stacks!